tests/e2e/convolution/CMakeLists.txt

```diff
@@ -323,3 +323,176 @@ iree_generated_e2e_runner_test(
 )
 
 ### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
+
+# To distinguish between CDNA(gfx9) and RDNA3(gfx11)
+if(IREE_HIP_TEST_TARGET_CHIP MATCHES "^gfx9")
+
+unset(IREE_HIP_TEST_COMPILER_FLAGS)
+list(APPEND IREE_HIP_TEST_COMPILER_FLAGS
+  "--iree-rocm-target-chip=${IREE_HIP_TEST_TARGET_CHIP}"
+)
```

Comment:
FYI I made it possible for some of this logic to be handled in the BUILD.bazel file in https://github.com/iree-org/iree/pull/17843. I didn't look specifically at folders like tests/e2e/matmul though. Generally speaking, while still using Bazel and CMake for tests, we should avoid using the `BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE` escape hatch if possible.